src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+
+import styles from './Slider.module.scss';
+import classNames from 'classnames/bind';
+const cx = classNames.bind(styles);
+
+import { Root as Root, Track as Track, Range as Range, Thumb as Thumb } from '@radix-ui/react-slider';
+
+export interface SliderProps {
+  step?: number;
+  min?: number;
+  max?: number;
+  value?: number;
+  className?: string;
+  minStep?: number;
+  isLight?: boolean;
+  isSmall?: boolean;
```

Comment:
Sorry I missed this one on the first review - can we change this to `size: 'default' : 'small'`. The prop should default to `default`, i.e.
```ts
export const Slider = ({
    ...
    size = 'small',
    ...
})
```

This keeps us open to possible design changes in the future, for example if they add a `large` variant.

Reply:
@colbr Ive just pushed up my interpretation of this, hope its right, otherwise lemme know :D